src/views/frontend/blocks/search/result.tpl

```diff
@@ -1,4 +1,13 @@
 <div class="boxwrapper">
+
+    [{assign var="oConfig" value=$oViewConf->getConfig()}]
+
+    [{if $oConfig->getConfigParam("ffCampaigns")}]
+        [{block name="ff_campaign_feedback_top"}]
```

Comment:
making this two blocks having same name `ff_campaign_feedback_top` will not give us benefit of overwriting only one.  In fact  i'm not sure if this is not an incorrect setup to have two blocks with the same name

Reply:
I believe it's incorrect setup, I forgot to rename second block, now it fixed